drivers/virtio/net/virtio_net.c

```diff
@@ -1007,6 +1043,16 @@ static int virtio_netdev_probe(struct uk_netdev *n)
 	 */
 	vndev->vdev->features = drv_features;
 
+	if (vndev->vdev->features & (1ULL << VIRTIO_F_VERSION_1)
```

Comment:
I would place paranthesis around both these bitwise `&`'s just to be double sure :man_shrugging:.